packages/cli/src/commands/migrate/tasks/config-lint.ts

```diff
@@ -60,12 +64,25 @@ async function extendsRehearsalInCurrentConfig(
   rehearsalConfigRelativePath: string,
   basePath: string = process.cwd()
 ): Promise<void> {
-  /* eslint-disable-next-line @typescript-eslint/no-var-requires */
-  const oldConfig = require(configPath);
-  const newConfig = {
-    ...oldConfig,
-    extends: Array.from(new Set([...oldConfig.extends, rehearsalConfigRelativePath])),
-  };
+  const projectName = determineProjectName(basePath);
+  const explorerSync = cosmiconfigSync(projectName || '');
```

Comment:
Why do we need cosmicconfig? What was the bug this is resolving?

Reply:
Originally we use `require' to retrieve the existing `eslintrc.js` in the target project. This works if we run `migrate` in the target project, but when we run `migrate` in a different directory and pass in `-p`, `require` does not work.